vaadin-checkbox-flow-parent/vaadin-checkbox-testbench/src/main/java/com/vaadin/flow/component/checkbox/testbench/CheckboxGroupElement.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.checkbox.testbench;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import com.vaadin.flow.component.checkbox.testbench.CheckboxElement;
+import com.vaadin.testbench.HasHelper;
+import com.vaadin.testbench.TestBenchElement;
+import com.vaadin.testbench.elementsbase.Element;
+
+@Element("vaadin-checkbox-group")
+public class CheckboxGroupElement extends TestBenchElement
+        implements HasHelper {
+
+    /**
+     * Gets a list of all available options.
+     *
+     * @return a list of the options (visible text)
+     */
+    public List<String> getOptions() {
+        return getCheckboxes().stream().map(CheckboxElement::getLabel)
+                .collect(Collectors.toList());
+    }
+
+    /**
+     * Gets all checkboxes which are part of this group.
+     *
+     * @return a list of all checkboxes in this group
+     */
+    public List<CheckboxElement> getCheckboxes() {
+        return $(CheckboxElement.class).all();
+    }
+
+    /**
+     * Selects/unselects the first checkbox matching the given text.
+     *
+     * @param text
+     *            the text of the option to select
+     */
+    public void changeSelectionByText(String text) {
```

Comment:
Consider using `selectByText` instead to align with other components e.g. `ListBoxElement`.

Reply:
I did it like that first, but then changed my mind as CheckBoxGroup is a multiselect, i.e. in ListBox and RadioButtonGroup it is naturally implicit that selectByText selectss new items, and unselects the previous one. In case of CheckBoxGroup selecting adds to selection, and I wanted to reflect that in the naming.

Reply:
I understand that, however in `MultiSelectComboBoxElement` we also use `selectByText` for consistency.

Reply:
`MultiSelectComboBoxElement `has also `unselectByText`, so if going that way, it should be added too.

Reply:
Yes, I'd say it makes sense. @yuriy-fix WDYT about this API?

Reply:
I updated the PR